applications/luci-app-advanced-reboot/root/usr/libexec/rpcd/luci.advanced_reboot

```diff
@@ -122,7 +126,8 @@ obtain_device_info(){
 	json_load_file "$p"
 	for i in vendorName deviceName partition1MTD partition2MTD labelOffset \
 		bootEnv1 bootEnv1Partition1Value bootEnv1Partition2Value \
-		bootEnv2 bootEnv2Partition1Value bootEnv2Partition2Value; do
+		bootEnv2 bootEnv2Partition1Value bootEnv2Partition2Value \
+		opOffset ubiVolume; do
 		json_get_var $i "$i"
```

Comment:
Have you tested this with opOffset and ubiVolume missing from json file?

Reply:
Yep, it behaves like it did before the changes, judging by logs. Looks like `json_get_var` can handle missing entries

Still, it'd be best to test on other supported routers before merging